### PR TITLE
Work with animations

### PR DESCRIPTION
**Pull request by Eduard-Kozachenko**
Add Material Design
-  https://trello.com/c/1qex71ds/4-%D0%B4%D0%BE%D0%B1%D0%B0%D0%B2%D0%B8%D1%82%D1%8C-material-design

Add alert dialog internet connection 
-  https://trello.com/c/mjgxDfvT/1-alert-dialog

Breed transfer to tool
- https://trello.com/c/3rV2BjHO/3-%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D1%87%D0%B0-%D0%BF%D0%BE%D1%80%D0%BE%D0%B4%D1%8B-%D0%B2-tool

Add Fragment transition animation
- https://trello.com/c/D1Xj7Diq/6-implement-android-navigation-component-and-add-fragment-transition-animation

Button back in toolbar
- https://trello.com/c/AzUbGKAj/2-%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B0-back

### CODE REVIEW

File: app/src/main/java/com/eduard/dogs/fragment/DogsListFragment.java [hunk 1]
```diff
@@ -87,13 +88,16 @@ public void onResponse(Call<JsonObject> call, Response<JsonObject> response) {
                                     }
                                 }
                                 listDataChild.put(listDataHeader.get(i), subBreedS);
-
                             }
 
+                            LayoutAnimationController controller = null;
```
Comment (vladislav-v-rusakov) [inline app/src/main/java/com/eduard/dogs/fragment/DogsListFragment.java]: Reduce to one line

File: app/src/main/res/layout/alert_inet.xml [hunk 2]
```diff
@@ -21,23 +21,23 @@
         android:text="@string/error_inet"
         android:id="@+id/text_dialog"
         android:layout_below="@+id/a"
-        android:layout_marginTop="20dp"
-        android:layout_marginLeft="4dp"
-        android:layout_marginRight="4dp"
-        android:layout_marginBottom="20dp"
-        android:textSize="18sp"
+        android:layout_marginTop="@dimen/margin_veinte"
```
Comment (vladislav-v-rusakov) [inline app/src/main/res/layout/alert_inet.xml]: Use English. 
Or numbers "@dimen/margin_20"

File: app/src/main/java/com/eduard/dogs/fragment/DogsListFragment.java (hunk 1 above)
Comment (Eduard-Kozachenko) [reply app/src/main/java/com/eduard/dogs/fragment/DogsListFragment.java]: ready

File: app/src/main/res/layout/alert_inet.xml (hunk 2 above)
Comment (Eduard-Kozachenko) [reply app/src/main/res/layout/alert_inet.xml]: ready

**Review by vladislav-v-rusakov — APPROVED** [review]

**Pull request merged**